src/keen-slider.js

```diff
@@ -166,9 +166,9 @@ function KeenSlider(initialContainer, initialOptions = {}) {
       e.preventDefault()
     })
     eventAdd(container, 'mousedown', eventDragStart)
-    eventAdd(container, 'mousemove', eventDrag)
-    eventAdd(container, 'mouseleave', eventDragStop)
-    eventAdd(container, 'mouseup', eventDragStop)
+    eventAdd(options.cancelOnLeave ? container : window, 'mousemove', eventDrag)
+    if (options.cancelOnLeave) eventAdd(container, 'mouseleave', eventDragStop)
+    eventAdd(window, 'mouseup', eventDragStop)
```

Comment:
Wondering if this change is intentional? this is changing the existing behavior as the `mouseup` event now binding to window regardless of  `options.cancelOnLeave` option, which previously bind to the `container`.

Some of our use case which manually handles `tap` (mouseup) on the slide and programmatically calls slider.next/slider.prev (v5), and this behavior breaks the use case.